cram-tests/command_help.t

```diff
@@ -0,0 +1,4 @@
+Test for attc --help command
+  $ attc --help > /dev/null
```

Comment:
On Linux, `attc --help` prints an X11 error when you redirect the output.  Not entirely sure why that is, but while we figure it out, I would recommend redirecting both `stderr` and `stdout` to `/dev/null`.  Something like this, I think?

```
attc --help >> /dev/null 2>&1
```
@waclena, please feel free to amend my shell redirecting code where applicable.

Reply:
Changed this to just `attc --help` as discussed.